configs/webserv_all_features.conf

```diff
@@ -29,9 +28,15 @@ http {
     # Define the max request body size, default is 1M
     client_max_body_size 1M;
 
+    # Define the default http request content type for all the servers, e.g. text/html, text/plain, application/json, etc.
+    default_type text/html;
+
     # Define the error page for the all the servers
     error_page 404 /404.html;
-    error_page 500 502 503 504 /50x.html;
+    error_page 500 /50x.html;
+    error_page 502 /50x.html;
+    error_page 503 /50x.html;
+    error_page 504 /50x.html;
     error_page 403 /403.html;
     error_page 401 /401.html;
```

Comment:
Why do we have 403 and 401 for server errors here?
*401: Unauthorized: The client must authenticate itself to get the requested response.
*403: Forbidden: The client does not have permission to access the requested resource.